client/app/components/sections/Topbar/Topbar.js

```diff
@@ -151,7 +162,10 @@ class Topbar extends Component {
             null;
     const mobileMenuAvatarProps = this.props.avatarDropdown && loggedInUsername ?
             { ...this.props.avatarDropdown.avatar, ...{ url: profileRoute } } :
-          null;
+            null;
+    const isAdmin = this.props.avatarDropdown && loggedInUsername ?
+            this.props.avatarDropdown.isAdmin :
```

Comment:
I suggest taking `isAdmin` out of `avatarDropdown` to root level, it might get confusing when components are interested in each other's properties. You can then set `isAdmin` for `avatarDropdown` here: https://github.com/sharetribe/sharetribe/pull/2244/files#diff-dadeb88347d45d99fce790d221cef12aR28